lib/concurrent-ruby/concurrent/utility/processor_counter.rb

```diff
@@ -181,13 +181,14 @@ def self.physical_processor_count
   end
 
   # Number of processors cores available for process scheduling.
-  # Returns `nil` if there is no #cpu_quota, or a `Float` if the
-  # process is inside a cgroup with a dedicated CPU quota (typically Docker).
+  # Returns number of processors seen by the OS if there is no #cpu_quota, or
+  # the value considering CPU quota if the process is inside a cgroup with a
+  # dedicated CPU quota (typically Docker).
```

Comment:
It seems hard to read given we describe the condition reversed, like if we explain the `else` part before the `then` part.
How about?
```suggestion 
  # This method takes in account the CPU quota if the process is inside a cgroup with a
  # dedicated CPU quota (typically Docker).
  # Otherwise it returns the same value as #processor_count but as a Float.
```

Reply:
Thanks for your review. 
I applied your suggestion.